openpype/hosts/maya/plugins/publish/extract_look.py

```diff
@@ -450,7 +515,7 @@ def resource_destination(self, instance, filepath, do_maketx):
             resources_dir, basename + ext
         )
 
-    def _process_texture(self, filepath, do_maketx, staging, linearize, force):
+    def _process_texture(self, filepath, do_rstex, do_maketx, staging, linearize, force):
```

Comment:
some as comment above

```suggestion
    def _process_texture(self, filepath, processors, staging, linearize, force):
```

Reply:
and you should iterate over processors on file path there

Reply:
Done!